kubehub/migrations/0002_auto_20200128_1204.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.0.2 on 2020-01-28 12:04
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('kubehub', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='cloudprovider',
+            name='cp_type',
+            field=models.CharField(choices=[('PROX_MOX', 'PROX_MOX'), ('AWS', 'AWS'), ('GCP', 'GCP')], max_length=20),
```

Comment:
for public view change PROX_MOX to Proxmox?

Reply:
fixed in https://github.com/dedicatted/kubehub-backend/pull/6       415f50a